src/libs/OptionsListUtils.js

```diff
@@ -708,7 +708,7 @@ function getOptions(
                 reportOption.isPolicyExpenseChat && reportOption.ownerEmail === currentUserLogin && includeOwnedWorkspaceChats && !reportOption.isArchivedRoom;
 
             // Skip if we aren't including multiple participant reports and this report has multiple participants
-            if (!isCurrentUserOwnedPolicyExpenseChatThatCouldShow && !includeMultipleParticipantReports && !reportOption.login) {
+            if (!isCurrentUserOwnedPolicyExpenseChatThatCouldShow && !includeMultipleParticipantReports && !reportOption.accountID) {
```

Comment:
what if we "play it safe" here and do something like:
```suggestion
            if (!isCurrentUserOwnedPolicyExpenseChatThatCouldShow && !includeMultipleParticipantReports && (!reportOption.login || !reportOption.accountID)) {
```

so that we support both cases since we're in the middle of a migration between the personal_details and personal_details_list? and do this everywhere `reportOption.login` is in this method?